package.json

```diff
@@ -18,10 +18,11 @@
     "test:watch": "jest --watch",
     "test:cov": "jest --coverage",
     "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
-    "test:e2e": "jest --config ./test/jest-e2e.json"
+    "test:e2e": "jest --config ./jest-e2e.json",
+    "test:e2eContainer": "docker build -t flatlify-tests . && sudo docker run flatlify-tests"
```

Comment:
- remove `sudo`
- rename `test:e2eContainer:start`
- let's make `test:e2eContainer:start` run as daemon - https://docs.docker.com/engine/reference/run/#detached--d
- let's create `test:e2eContainer:stop` to stop the container

Reply:
Let's discuss scrips names